src/index.html

```diff
@@ -17,11 +17,423 @@
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="img imac"
+          class="card__img"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__review">Reviews: 5</p>
+        </div>
+
+        <p class="price">
+          <span class="price__title">Price:</span>
+
+          <span class="price__money">$2,199</span>
+        </p>
+
+        <a
+          href="#"
+          class="button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="img imac"
+          class="card__img"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__product">Product code: 195434</p>
+
+        <div class="reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="reviews__review">Reviews: 5</p>
+        </div>
+
+        <p class="price">
+          <span class="price__title">Price:</span>
+
+          <span class="price__money">$2,199</span>
+        </p>
+
+        <a
+          href="#"
+          class="button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div
```